d3a_interface/device_validator.py

```diff
@@ -60,9 +61,34 @@ def validate_energy(cls, **kwargs):
     @classmethod
     def validate_rate(cls, **kwargs):
         """Validate rates of a Home Meter device."""
+
+        cls._validate_fit_to_limit(
+            fit_to_limit=kwargs.get("fit_to_limit"),
+            energy_rate_increase_per_update=kwargs.get("energy_rate_increase_per_update"),
+            energy_rate_decrease_per_update=kwargs.get("energy_rate_decrease_per_update"))
         cls._validate_home_meter_consumption_rates(**kwargs)
         cls._validate_home_meter_production_rates(**kwargs)
 
+    @staticmethod
+    def _validate_fit_to_limit(
+            fit_to_limit: Optional[bool], energy_rate_increase_per_update: Optional[bool],
+            energy_rate_decrease_per_update: Optional[bool]):
+        if fit_to_limit is True and (
```

Comment:
What other value can `fit_to_limit` hold besides True, False or None? if no other values can be received remove the is True operator

Reply:
In general I agree with you. In this specific case I didn't know whether we should have been more strict (in case someone decided to pass `fit_to_limit="yeah!"`) but I just remembered that the schema should fail in that case: https://github.com/gridsingularity/d3a-interface/blob/348a9ecb626e23cbeff2694ebc84a587b1f730dd/d3a_interface/schemas.py#L62

So I can change it as you suggested. Thanks!

Reply:
Please let me remind you on the frontend. They there are a lot of places where they send "null" instead of a False, so please prepare for that too.

Reply:
Sorry, only got it now. So all good.